libs/components/src/accordion/stories/vl-accordion.stories-arg.ts

```diff
@@ -59,6 +60,15 @@ export const accordionArgTypes: ArgTypes<typeof accordionArgs> = {
             defaultValue: { summary: accordionArgs.disabled },
         },
     },
+    isOpen: {
+        name: 'data-vl-is-open',
```

Comment:
gezien je `data-vl-is-open` toevoegt, is er hier geen mogelijkheid om het gewoon reactief te maken? heb de code niet goed genoeg bekeken in welke mate dit technisch mogelijk zou zijn

Concreet zou dit willen zeggen dat:
- het attribuut die `open` voorstelt, er automatisch wordt afgehaald wanneer het gesloten wordt & vice versa.
- maar ook wanneer de afnemer `data-vl-is-open`-attribuut verwijdert, dat het ook zal sluiten.

Reply:
Ik zag het nut niet echt in. Als het dient om de eerste state in te stellen hoeft dat niet imo. Anders zou je gewoon open en close gewoon kunnen aanroepen

Reply:
Al is de attribuutnaam dan inderdaad beetje ongelukkig.
Data-vl-is-standaard-open ?

Reply:
(voorlopig gewijzigd naar "is-standaard-open")

Reply:
Bedoelde "standaard" voor beschrijving/description. Dan is "is-open" wel beter.

Attributen zelf en code houden we doorgaans in het Engels. Spijt me voor de verwarring. Kunnen straks ook irl bespreken.